walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -268,6 +269,108 @@ public void notEnoughRoom() {
     Collection<TimeRange> actual = query.query(events, request);
     Collection<TimeRange> expected = Arrays.asList();
 
+    Assert.assertEquals(expected, actual);
+  }
+  @Test
+  public void optionalBusyAllDay() {
```

Comment:
Can you please add a test case with following events:
Events: 
PERSON_A: whole day
PERSON_B: 8-8:30a
PERSON_B: 9-9:30a
PERSON_B: 10-10:30a

Reply:
Huh, didn't catch this bug, thanks!